generic-styles/index.less

```diff
@@ -143,14 +151,16 @@ body {
   sub {
     bottom: -.25em;
   }
-  h2{
-    font-size: 3.2rem;
+  h2 {
+    .content-font-size(3.2rem);
+    line-height: calc(3.5rem * var(--content-text-scale));
   }
   h3 {
-    font-size: 2.4rem;
+    .content-font-size(2.4rem);
+    line-height: calc(2.3rem * var(--content-text-scale));
```

Comment:
you wanna make a similar helper for line height?

Reply:
yeah good idea, at first there were only 2 entries and didn't think about it after we updated a few other places